### ISSUE TITLE
Section : how to well train a bert model

### ISSUE THREAD

**Issue by emilienschultz** [activetigger/documentation#23]
What is the general logic of fine-tuning a bert model, how to choose a model, how many annotation do we need, how to read the loss, how to iterate, which metric to use, how to decide the model is good
